tests/test_architecture.py

```diff
@@ -43,6 +43,7 @@ def test_diagram_package():
         .may_import("gaphor.diagram*")
         .may_import("gaphor.UML.recipes")
         .may_import("gaphor.UML.uml")
+        .may_import("gaphor.ui.*")
```

Comment:
Let's keep this for now.

I consider this a regression, though. But it will require a bit more work internally to restructure code ina way that we can use something like a file dialog from the diagram module too.